cmd/ui/src/views/DatabaseManagement/DatabaseManagement.tsx

```diff
@@ -222,7 +222,7 @@ const DatabaseManagement = () => {
     return (
         <ContentPage title='Clear BloodHound Data'>
             <Box>
-                <Typography variant='body1'>
+                <Typography variant='body2'>
```

Comment:
These helper texts seem like they are sub headers to the page. I would recommend using the component prop with the typography component to render these with elements that are more semantically appropriate like an h2. It might even make sense to update the ContentPage component to accept props for different headers and handle the rendering there similar to the `title` prop that it currently uses. 

Disregard if you feel that these helper texts should not be considered headers.

Reply:
The intention is to provide a basic description and/or help links to guide users to what to expect. Usually, I expect an H2 to be a title for some specific section of content on the page. So while I'm not sure a sub-header is correct in this case, perhaps this is a new `pageDescription` prop that automatically puts the content in a box with a consistent variant instead of applying `body2` on each page?

Reply:
Yea I see what you're saying and agree. I may have been envisioning something like 
```html
<header>
  <h1>title</h1>
  <p>page description</p>
</header>
```
so what you are describing with `pageDescription` makes sense.